src/main/java/uk/gov/hmcts/reform/hmc/constants/Constants.java

```diff
@@ -10,4 +10,5 @@ private Constants() {
         + "your browser or try again later";
     public static final String INVALID_ERROR_CODE_ERR_MESSAGE = "Error code is invalid";
     public static final String INVALID_HEARING_PAYLOAD = "Invalid json request";
+    public static final String INVALID_LOCATION_REFERENCES = "Location reference EPIMS must be supplied";
```

Comment:
Maybe..

```suggestion
    public static final String INVALID_LOCATION_REFERENCES = "At least one EPIMS location reference must be supplied";
```

Reply:
Fixed in https://github.com/hmcts/hmc-hmi-inbound-adapter/pull/57/commits/d6c7102ac5ec829df9b005bb47e9e623bb26f9f9